gulpfile.js

```diff
@@ -1,32 +1,87 @@
+// var gulp = require('gulp'),
```

Comment:
Why is this entire area being commented and left behind? It should be removed if it's no longer part of the Gulp build process.